build.gradle

```diff
@@ -1,4 +1,5 @@
 buildscript {
+    ext.kotlin_version = '1.3.21'
```

Comment:
Yes, since its needed both in this file for `kotlin-gradle-plugin` and in build.gradle of Vision project. And here we cannot depend on external versions.gradle =\